uxarray/plot/dataarray_plot.py

```diff
@@ -61,3 +71,159 @@ def datashade(uxda: UxDataArray,
         _cmap = cmap
 
     return tf.shade(aggregated, cmap=_cmap, **kwargs)
+
+
+def _point_raster(uxda: UxDataArray,
+                  backend: Optional[str] = "bokeh",
+                  pixel_ratio: Optional[float] = 1.0,
+                  dynamic: Optional[bool] = False,
+                  precompute: Optional[bool] = True,
+                  projection: Optional[ccrs] = None,
+                  width: Optional[int] = 1000,
+                  height: Optional[int] = 500,
+                  colorbar: Optional[bool] = True,
+                  cmap: Optional[str] = "Blues",
+                  aggregator: Optional[str] = "mean",
+                  interpolation: Optional[str] = "linear",
+                  npartitions: Optional[int] = 1,
+                  **kwargs):
+
+    if uxda.face_centered():
+        # data mapped to face centroid coordinates
+        data_mapping = "center"
+        lon = uxda.uxgrid.Mesh2_face_x.values
+        lat = uxda.uxgrid.Mesh2_face_y.values
+    elif uxda.node_centered():
+        # data mapped to face corner coordinates
+        data_mapping = "corner"
+        lon = uxda.uxgrid.Mesh2_node_x.values
+        lat = uxda.uxgrid.Mesh2_node_y.values
+    else:
+        raise ValueError(
+            "Issue with data. It is neither face-centered nor node-centered!")
+
+    data_values = uxda.values
+
+    recompute = True
+    if data_mapping == "center":
+        if uxda.uxgrid._centroid_points_df_proj[
+                0] is not None and uxda.uxgrid._centroid_points_df_proj[
+                    1] == projection:
+            recompute = False
+            points_df = uxda.uxgrid._centroid_points_df_proj[0]
+
+    else:
+        if uxda.uxgrid._corner_points_df_proj[
+                0] is not None and uxda.uxgrid._corner_points_df_proj[
+                    1] == projection:
+            recompute = False
+            points_df = uxda.uxgrid._corner_points_df_proj[0]
+
+    if recompute:
+        # need to recompute points & projection
+        if projection is not None:
+            lon, lat, _ = projection.transform_points(ccrs.PlateCarree(), lon,
+                                                      lat).T
+
+        point_dict = {"lon": lon, "lat": lat, "var": data_values}
+
+        # Construct Dask DataFrame
+        point_ddf = dd.from_dict(data=point_dict, npartitions=npartitions)
+
+        points = hv.Points(point_ddf, ['lon', 'lat'])
+
+        # cache computed points & projection
+        if data_mapping == "center":
+            uxda.uxgrid._centroid_points_df_proj[0] = point_ddf
+            uxda.uxgrid._centroid_points_df_proj[1] = projection
+        else:
+            uxda.uxgrid._corner_points_df_proj[0] = point_ddf
+            uxda.uxgrid._corner_points_df_proj[1] = projection
+
+    else:
+        # can use existing cached points & projection
+        points_df['var'] = pd.Series(data_values)
+        points = hv.Points(points_df, ['lon', 'lat'])
+
+    if backend == "matplotlib":
```

Comment:
It seems like, when backend is MPL, we are not using the width and height provided. Shouldn't we?

Reply:
The MPL backend does not accept width/height arguments. It does, however, accept a `fig_size` which is the closest similar argument.

https://holoviews.org/user_guide/Plotting_with_Matplotlib.html#plot-size

Reply:
Yeah, do you think we should take these optional args in as "hv_width, hv_height", and "mpl_figsize" and document which one is for what backend?

Reply:
That could work, but I don't like the "hv" or "mpl" prefix in front of these variables. In relativity, any backend-specific arguments should be handled by `**kwargs`. 

We could keep the `height` and `width` the same, include a small blurb in it's docstring, and introduce a default `fig_size` . What do you think?

Reply:
Sounds good to me.

Reply:
Great!